crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -219,7 +218,7 @@ pub fn check_visibility(
                     let model = transform.compute_matrix();
                     let model_sphere = Sphere {
                         center: model.transform_point3a(model_aabb.center),
-                        radius: (Vec3A::from(transform.scale) * model_aabb.half_extents).length(),
+                        radius: transform.radius(model_aabb.half_extents.into()),
```

Comment:
radius_vec3a perhaps. I chose Vec3A here as it has a performance benefit over Vec3 as it uses SIMD, which is particularly important in this function.

Reply:
Agreed!